src/main/java/io/cryostat/jfr/datasource/server/JfrResource.java

```diff
@@ -130,6 +139,18 @@ void list(RoutingContext context) {
         }
     }
 
+    @Route(path = "/current")
+    void current(RoutingContext context) {
+        HttpServerResponse response = context.response();
+        setHeaders(response);
+
+        StringBuilder responseBuilder = new StringBuilder();
+        System.out.println(loadedFile);
+        responseBuilder.append(loadedFile);
+        responseBuilder.append(System.lineSeparator());
+        response.end(responseBuilder.toString());
```

Comment:
The `StringBuilder` isn't really necessary here. It's beneficial when you're constructing a `String` by looping over elements of a collection, for example, but here it's always just one element followed by a line separator. You can just do "response.end(loadedFile + System.lineSeparator())` in that case.